src/test/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurerTests.java

```diff
@@ -241,6 +241,18 @@ public void testIsStringAlreadyPresentInFile_isPresent() throws IOException {
         assertThat(isKeyPresentInYMLFile(installer.OPENSEARCH_CONF_FILE, str2), is(equalTo(false)));
     }
 
+    @Test
+    public void testAssumeYesDoesNotInitializeClusterMode() throws IOException {
+        String str1 = "node.name"; // cluster_mode
+        String str2 = "plugins.security.allow_default_init_securityindex"; // init_security
+
+        installer.assumeyes = true;
+        securitySettingsConfigurer.writeSecurityConfigToOpenSearchYML();
+
+        assertThat(isKeyPresentInYMLFile(installer.OPENSEARCH_CONF_FILE, str1), is(equalTo(false)));
+        assertThat(isKeyPresentInYMLFile(installer.OPENSEARCH_CONF_FILE, str2), is(equalTo(false)));
```

Comment:
```suggestion
        assertThat(isKeyPresentInYMLFile(installer.OPENSEARCH_CONF_FILE, str1), is(false));
        assertThat(isKeyPresentInYMLFile(installer.OPENSEARCH_CONF_FILE, str2), is(false));
```

Reply:
Done in: [60769b3](https://github.com/opensearch-project/security/pull/3935/commits/60769b3e2461377e9d0c459e519ecc9ad0a35762)